### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jjaykim**
# Befor putting PR (put x in bracket if you ran it.);

- [x] Run `pylint --rcfile=.pylintrc $(git ls-files '*.py')` passes

# Overview

Create a website for Poro Bot.

### CODE REVIEW

**Comment by dhmoon91** [comment]
post screenshots plz

**Comment by jjaykim** [comment]
> post screenshots plz

**Desktop Version**
Desktop-Version-1
<hr/>
Screen Shot 2021-08-25 at 3 22 48 PM

<hr/>
Desktop-Version-3

**Mobile Version**
Mobile-Version-1
<hr/>
Screen Shot 2021-08-25 at 3 23 24 PM
<hr/>
Mobile-Version-3
<hr/>
Mobile-Version-4

작동하는 모습을 녹화했는데 용량이 커서 그건 디스코드에 `web` 체널에 업로드하겠습니다 !

File: web/src/components/about/About.jsx [hunk 1]
```diff
@@ -2,35 +2,95 @@ import React from 'react';
...
+                </div>
+
+                <div className="d-flex flex-column align-items-center text-center">
+                  <p>Used By</p>
+                  <h4>Over 1,000 Servers</h4>
+                </div>
+
+                <div className="d-flex flex-column align-items-center text-center">
+                  <p>Used By</p>
+                  <h4>Over 1,000 Channels</h4>
```
Comment (dhmoon91) [inline web/src/components/about/About.jsx]: we gotta update this all 3 sections using same text

File: web/src/components/about/About.jsx [hunk 2]
```diff
@@ -2,35 +2,95 @@ import React from 'react';
...
+                </div>
+
+                <div className="d-flex flex-column align-items-center text-center">
+                  <p>Used By</p>
+                  <h7>Over 1,000 Servers</h7>
+                </div>
+
+                <div className="d-flex flex-column align-items-center text-center">
+                  <p>Used By</p>
+                  <h7>Over 1,000 Channels</h7>
```
Comment (dhmoon91) [inline web/src/components/about/About.jsx]: same here

File: web/src/components/header/Header.jsx [hunk 3]
```diff
@@ -33,31 +38,103 @@ const Header = () => {
...
-        <Container>
-          <Navbar.Brand href="#home">
-            <div className="d-flex justify-content-between align-items-center">
-              <img alt="Poro logo" src={Poro} width="50" height="50" className="d-inline-block align-top" />
-              <div className="text-white">Poro</div>
-            </div>
-          </Navbar.Brand>
-          <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+      <Default>
+        <div>
```
Comment (dhmoon91) [inline web/src/components/header/Header.jsx]: u already wrapped this in <Default> is wrapping it in `<div>` again necessary?

File: web/src/components/header/Header.jsx [hunk 4]
```diff
@@ -33,31 +38,103 @@ const Header = () => {
...
+                    <Nav.Link className="get-porobot" onClick={() => window.open(ADD_URL, '_blank')}>
+                      Get PoroBot
+                    </Nav.Link>
+                  ) : null}
+                </Nav>
+              </Navbar.Collapse>
+            </Container>
+          </Navbar>
+        </div>
+      </Default>
```
Comment (dhmoon91) [inline web/src/components/header/Header.jsx]: If you are going to use `Default` and `Mobile` components, it's better to put these out into separate components. 
eg; 
create new; HeaderDesktop.jsx and HeaderMobile.jsx and this should just import those components

File: web/src/components/about/About.jsx (hunk 1 above)
Comment (jjaykim) [reply web/src/components/about/About.jsx]: `using same text` 가 3개 sections 모두 `over 1,000` 똑같아서 숫자를 바꾸라는 말씀이신가요?
아니면 `Desktop`일 땐 `h4`이고 `Mobile`일 땐 `h7`이여서 같은 tag를 쓰라는 말씀이신가요??

Comment (dhmoon91) [reply web/src/components/about/About.jsx]: 아니 지금 3개다 
`used by over 1,000 channels` 잖아 ㅋㅋㅋㅋ
3개가 그래도 좀 틀려야지. 

1) `Used by over 100 channels`
2) `User friendly commands`
3)은 모 너나 낙일이랑 생각해서 써봐

Comment (jjaykim) [reply web/src/components/about/About.jsx]: 아 ㅋㅋㅋㅋㅋ 네네 !

**Review by jung1411 — APPROVED** [review]

**Comment by dhmoon91** [comment]
what is this  web/package-lock 2.json why do you have package-lock `2` file??

File: web/src/components/footer/Footer.jsx [hunk 5]
```diff
@@ -12,32 +14,48 @@ const Footer = () => {
...
           <div className="d-flex justify-content-around w-75 text-white">
-            <div className="d-flex flex-column">
-              <h5>Block header</h5>
-              <p>Block content</p>
+            <div className="w-50">
+              <p>
+                If there is any issues or have any questions regarding our bot, please mention in our{' '}
+                <a
+                  className="server-link-text"
+                  href="https://discord.gg/AZe8ETHV"
```
Comment (dhmoon91) [inline web/src/components/footer/Footer.jsx]: what is this link?

Comment (dhmoon91) [reply web/src/components/footer/Footer.jsx]: @jjaykim @jung1411 What is this link??? Did you guys create new server?

**Comment by dhmoon91** [comment]
> what is this web/package-lock 2.json why do you have package-lock `2` file??

@jjaykim @jung1411 I see it's deleted but why was this created?? Please lmk

File: web/src/components/footer/Footer.jsx (hunk 5 above)
Comment (jung1411) [reply web/src/components/footer/Footer.jsx]: yeah, we created one for answering questions and stuff and made the invite link permanent

Comment (jung1411) [reply web/src/components/footer/Footer.jsx]: > > what is this web/package-lock 2.json why do you have package-lock `2` file??
> 
> @jjaykim @jung1411 I see it's deleted but why was this created?? Please lmk

I think we were sending the folders manually back and forth during the beginning of the website creation and  I think it somehow got duplicated at that time. Then we created another repo afterwards just for me and jay to work on for the website part. I think we were just kind of confused on how we should share and it was totally unnecessary. We should've just created issue/task for each components and pushed on to this repo.

**Comment by jung1411** [comment]
> > what is this web/package-lock 2.json why do you have package-lock `2` file??
> 
> @jjaykim @jung1411 I see it's deleted but why was this created?? Please lmk

I think we were sending the folders manually back and forth during the beginning of the website creation and I think it somehow got duplicated at that time. Then we created another repo afterwards just for me and jay to work on for the website part. I think we were just kind of confused on how we should share and it was totally unnecessary. We should've just created issue/task for each components and pushed on to this repo.

**Pull request merged**